### ISSUE TITLE
Identify focus styles and order

### ISSUE THREAD

**Issue by mkhandekar** [mozilla-rally/rally#362]
## Goal

Users should be able to use a keyboard and screenreaders to navigate Rally.

## Definition of Done

* Focus order is identified - use Stark [(figma plugin)](https://www.figma.com/community/plugin/732603254453395948/Stark)
  * How to: https://www.getstark.co/blog/how-your-landmarks-in-design-support-screen-reader-users/
* Focus styles are designed (as simple as possible)

## Tasks 
- [ ] Build ideal focus order for each user flow 
- [ ] Design focus style

cc: @tangcasey for visibility
